test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -1249,7 +1249,7 @@ public void TensorFlowImageClassification()
             var pipeline = mlContext.Transforms.LoadImages("Image", fullImagesetFolderPath, false, "ImagePath") // false indicates we want the image as a VBuffer<byte>
                 .Append(mlContext.Model.ImageClassification(
                     "Image", "Label",
-                    // Just by changing/selecting InceptionV3 here instead of 
+                    // Just by changing/selecting InceptionV3/MobilenetV2 here instead of 
```

Comment:
Do you want to add a test for the new functionality? The CI isn't executing any of your new code.

Reply:
Added unit test.

---
In reply to: [336686355](https://github.com/dotnet/machinelearning/pull/4351#discussion_r336686355) [](ancestors = 336686355)